### PR TITLE
Add FadableContent to convey trimmed code

### PR DESCRIPTION
**Pull request by kumar303**
Fixes https://github.com/mozilla/addons-code-manager/issues/959

Example from the storybook:

Screenshot 2019-07-18 11 33 06

### CODE REVIEW

**Comment by kumar303** [comment]
I noticed a bug here that I need to fix. Stay tuned...

File: src/components/FadableContent/styles.module.scss [hunk 1]
```diff
@@ -0,0 +1,14 @@
+@import '../../scss/variables';
+
+.shell {
+  position: relative;
+
+  &::after {
+    background-image: linear-gradient(rgba(255, 255, 255, 0), $white);
```
Comment (willdurand) [inline src/components/FadableContent/styles.module.scss]: I think using `transparent` would be a little more explicit than `rgba(...)`.

File: src/components/FadableContent/styles.module.scss [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    bottom: 0;
+    content: '';
+    height: 400px;
```
Comment (willdurand) [inline src/components/FadableContent/styles.module.scss]: Why is this enforced in this component? Could we configure its value somehow?

File: src/components/FadableContent/styles.module.scss (hunk 1 above)
Comment (kumar303) [reply src/components/FadableContent/styles.module.scss]: Safari handles "transparent" in a wacko way. See the first answer here: https://stackoverflow.com/questions/38391457/linear-gradient-to-transparent-bug-in-latest-safari . That's really the only reason I avoided it. I guess Safari doesn't matter? I always like to aim for cross-compat when possible.

File: src/components/FadableContent/styles.module.scss (hunk 2 above)
Comment (kumar303) [reply src/components/FadableContent/styles.module.scss]: I don't think anything would need to configure it but if that day comes we could easily add it to the component props.

File: src/components/FadableContent/styles.module.scss (hunk 1 above)
Comment (kumar303) [reply src/components/FadableContent/styles.module.scss]: This is what using `transparent` looks like in today's Safari on macOS Mojave

Screenshot 2019-07-19 10 12 32

Comment (kumar303) [reply src/components/FadableContent/styles.module.scss]: I can add a comment to the css about it

Comment (willdurand) [reply src/components/FadableContent/styles.module.scss]: oh mmh, ok then let's keep `rgba` and document it.

File: src/components/FadableContent/styles.module.scss (hunk 2 above)
Comment (willdurand) [reply src/components/FadableContent/styles.module.scss]: Why is it set to `400px` only then? Looks ridiculously small on my screen for example. Usually, we tend to consider the height of the window, no?

Comment (kumar303) [reply src/components/FadableContent/styles.module.scss]: Hmm. Could you send a screenshot? `400px` looked fine when viewing actual truncated code in `DiffView` or `CodeView`. I could switch it to `vh`, I suppose.

Comment (kumar303) [reply src/components/FadableContent/styles.module.scss]: I switched it to `50vh`

**Comment by kumar303** [comment]
Thanks for the review

**Pull request merged**